tests/e2e/cluster_downgrade_test.go

```diff
@@ -47,13 +48,19 @@ func testDowngradeUpgrade(t *testing.T, clusterSize int) {
 		t.Skipf("%q does not exist", lastReleaseBinary)
 	}
 
-	currentVersion := semver.New(version.Version)
-	lastVersion := semver.Version{Major: currentVersion.Major, Minor: currentVersion.Minor - 1}
+	currentVersion, err := getVersionFromBinary(currentEtcdBinary)
+	require.NoError(t, err)
+	lastVersion, err := getVersionFromBinary(lastReleaseBinary)
+	require.NoError(t, err)
 
-	currentVersion.PreRelease = ""
+	require.Equalf(t, lastVersion.Minor, currentVersion.Minor-1, "unexpected minor version difference")
 	currentVersionStr := currentVersion.String()
 	lastVersionStr := lastVersion.String()
 
+	lastVersionZero := semver.New(lastVersionStr)
+	lastVersionZero.Patch = 0
+	lastVersionZeroStr := lastVersionZero.String()
```

Comment:
```suggestion
	lastVersionZero := version.Cluster(lastVersionStr)
```

Reply:
that doesn't work quite as well:
> {"error": "expect etcdcluster version 3.5, but got 3.5.0"}

The cluster version splits the patch release off

Reply:
Ok, you might need to go through `semver.New().String()`, but let's use `version.Cluster` to get cluster version.